packages/tocco-ui-showcase/src/components/StyledShowCaseApp.js

```diff
@@ -0,0 +1,105 @@
+import styled, {injectGlobal} from 'styled-components'
+
+// eslint-disable-next-line chai-friendly/no-unused-expressions
+injectGlobal`
```

Comment:
are you sure about injectGlobal? 
The official website says: 
> We do not encourage the use of this. Try to use it once per app at most, if you must, contained in a single file. This is an escape hatch. Only use it for the rare @font-face definition or body styling.

Reply:
> definition or **body** styling

`injectGlobal` is the only way to declare styles on `<body>`

Current body styles are obsolete. Hence I have removed it.